### PR TITLE
Add CHANGELOG.rst and Minor CI fix

### PR DESCRIPTION
**Pull request by yukiisbored**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+.. _Semantic Versioning: https://semver.org/spec/v2.0.0.html
+
+*************
+`Unreleased`_
+*************
+
+Added
+=====
+
+* ``IMGPROXY_SKIP``, option to skip imgproxy URL generation and only
```
Comment (ksdme) [inline CHANGELOG.rst:20]: Does double backtick mean something or is this a typo?

Comment (yukiisbored) [reply CHANGELOG.rst:20]: In rST, single backtick means link while double backtick means verbatim/code.

Ref: https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html

Comment (ksdme) [reply CHANGELOG.rst:20]: 👍

**Comment by ksdme** [comment]
LGTM

**Pull request merged**